### PR TITLE
Part 7 - Classes

### PR DESCRIPTION
**Pull request by VinayakaMayura**

### CODE REVIEW

File: part-1/7-Classes/README.md [hunk 1]
```diff
@@ -1,2 +1,120 @@
...
+        this.model_name = model_name;
+        this.brand = brand
+        this.model = model;
+    }
+}
+
+class Car extends Vehicle {
+
+    noOfGears(){
+        let car_gears = 5;
```
Comment (saikrishna321) [inline part-1/7-Classes/README.md]: Change this to carGears

File: part-1/7-Classes/README.md [hunk 2]
```diff
@@ -1,2 +1,120 @@
...
+        return car_gears;
+    }
+
+    getVechicleDetails(){
+        let vechicleDetails = `${this.brand} ${this.model_name} ${this.model} ${this.noOfGears()} Gears`;
+        return vechicleDetails;
+    }
+
+    static getCarPrice() {
+        let car_price = "10 lakhs";
```
Comment (saikrishna321) [inline part-1/7-Classes/README.md]: Change to carPrice

File: part-1/7-Classes/README.md [hunk 3]
```diff
@@ -1,2 +1,120 @@
...
+
+Inheritance in Javascript can also be acheived using the Prototype.
+
+### Static Methods
+
+The `static` keyword defines function in a class as static function. `static` functions can be called without instantiating the class. 
+
+```javascript
+    static getCarPrice() {
+        let car_price = "10 lakhs";
```
Comment (saikrishna321) [inline part-1/7-Classes/README.md]: Same here

File: part-1/7-Classes/README.md [hunk 4]
```diff
@@ -1,2 +1,120 @@
...
+```
+
+## Inhertiance
+
+Class Inheritance is a way to extend from another class. New functionality can be added to the parent class.
+
+```javascript
+class Vehicle {
+    constructor(model_name, brand, model){
+        this.model_name = model_name;
```
Comment (saikrishna321) [inline part-1/7-Classes/README.md]: Same here: Should be modelName

**Review by saikrishna321 — CHANGES_REQUESTED** [review]

File: part-1/7-Classes/README.md (hunk 1 above)
Comment (VinayakaMayura) [reply part-1/7-Classes/README.md]: Updated

File: part-1/7-Classes/README.md (hunk 2 above)
Comment (VinayakaMayura) [reply part-1/7-Classes/README.md]: Updated

File: part-1/7-Classes/README.md (hunk 3 above)
Comment (VinayakaMayura) [reply part-1/7-Classes/README.md]: Updated

File: part-1/7-Classes/README.md (hunk 4 above)
Comment (VinayakaMayura) [reply part-1/7-Classes/README.md]: Updated

**Review by saikrishna321 — APPROVED** [review]

**Pull request merged**